qt/widgets/mplcpp/inc/MantidQtWidgets/MplCpp/Colormap.h

```diff
@@ -11,7 +11,11 @@
 #undef slots
 #include "MantidQtWidgets/Common/Python/Object.h"
 #pragma pop_macro("slots")
+#include <QPushButton>
```

Comment:
Why did you add all of these imports in this file? `Colormap.cpp` didn't change in this PR so I'm surprised that more imports are needd.